### ISSUE TITLE
Distance and Direction - create lines from distance and bearing broken

### ISSUE THREAD

**Issue by Dbarnes1** [Esri/solutions-webappbuilder-widgets#836]
when creating a line from distance and bearing, I'm unable to get the line to update when I click enter after entering a specific distance and degree bearing. Instead, all I am able to do is draw the tool interactively as if I were still drawing from points. See screenshot for result, and note that distance and bearing is the option selected: 
![image](https://cloud.githubusercontent.com/assets/13024357/23178361/fa4870de-f81e-11e6-93a6-8be3b8d2f7e2.png)

Not sure if this has something to do with @adgiles unit test refactoring, but I thought that this functionality had previously worked.

**Comment by adgiles** [comment]
@Dbarnes1 

Dan this issue should be resolved, I have slightly changed the behaviour of this part of the tool for it to make more sense to the user, if you select distance and bearing from the drop down the drawing tool changes from line input to point input. The user then drops a point on the map, types in a length and then an angle, hits return in the angle box and the line is drawn.

Switching the drop down back to points should reset the tool back to line input

Please can you verify all behaving as it should

**Comment by Dbarnes1** [comment]
@adgiles lines look good! I checked against the other shapes and it seems they are still working fine as well. The only thing that I'm noticing now is that the map doesn't zoom to the extent of the line output when it is created from distance and bearing. I think to mimic the distance calculator for circles and non-interactive range rings, this functionality should be added. I logged this enhancement request here: https://github.com/Esri/solutions-webappbuilder-widgets/issues/838
